### ISSUE TITLE
Will this only work with nvidia?

### ISSUE THREAD

**Issue by TornSky** [JoshuaKimsey/Linux-StableDiffusion-Script#17]
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

I have an AMD Card 

The card I have is : AMD Vega 20 radeon VII 

PS. Thanks for making this script It helps a ton!

![Screenshot from 2022-10-07 21-25-18](https://user-images.githubusercontent.com/105568636/194688021-58e33dba-aa0b-46fa-ba4f-a84d67e294a6.png)

**Comment by JoshuaKimsey** [comment]
Yeah, as of right now my script is only for the Nvidia version. There is a way to make this work with AMD, but you will need to ask on the actual SD-WebUI fork repo itself for how to do so.
